src/devices/Dhtxx/Devices/Dht21.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.Gpio;
+using Iot.Units;
+
+namespace Iot.Device.DHTxx
+{
+    /// <summary>
+    /// Temperature and Humidity Sensor DHT21
+    /// </summary>
+    public class Dht21 : DhtBase
```

Comment:
how's this code different from Dht12

Reply:
They are different sensors, and I listed the parameters in readme.